api/tests/test_security_user.py

```diff
@@ -127,3 +127,28 @@ def test_security_user_api_endpoint_cant_allow_delete(self, mock_verify_token):
             HTTP_AUTHORIZATION="Token {}".format(self.token_admin),
         )
         self.assertEqual(response.data, {'detail': 'Method "DELETE" not allowed.'})
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_admin_can_filter_security_users_by_status(self, mock_verify_id_token):
+        mock_verify_id_token.return_value = {'email': self.admin_user.email}
+        response = client.get(
+            '{}?active={}'.format(
+                self.security_users_admin_url, self.security_user.is_active
+            ),
+            HTTP_AUTHORIZATION="Token {}".format(self.token_admin),
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data['count'], 1)
+        self.assertEqual(response.data['results'][0]['email'], self.security_user.email)
```

Comment:
tiny last request - let's use `assertIn` to avoid this failing if more security users are added in setup

Reply:
okay